docs/software/serial-protocols.md

```diff
@@ -59,6 +61,43 @@ When connected to the receiver with the transmitter, enter the ExpressLRS Lua sc
 ![output mapping serial](../assets/screenshots/receiver-serial.png)
 </figure>
 
+## Second Serial Interface on ESP32 PWM receivers
+
+Starting with firmware version 3.5 an additional serial interface (Serial2) can be assigned on any featured PWM Output to run any of the available serial protocols. Both serial interfaces can be be utilized at the same. This enables setups like running a CRSF wing stabilization unit and HoTT telemetry, Tramp or SmartAudio at the same time. Serial2 can also be utilized to simplify the wiring of e.g. an SBUS connection on receivers which have the first serial interface on separate connectors (e.g. RM receivers with JST connectors).
```

Comment:
"can be utilized at the same time"? or use "simultaneously" as the next sentence is already using "same time".

Reply:
done